src/data.js

```diff
@@ -11336,6 +11336,35 @@ module.exports = [
       'C#',
     ],
   },
+  {
+    name: 'Derek Mohr',
+    description: 'Front-end development by interest, full stack development by trade',
+    url: 'https://onemohrti.me/uses/',
```

Comment:
Error: Ping to "https://onemohrti.me/uses/" failed with status: 404

Reply:
@gatarelib is there a way to run a re-check? It's definitely there, I wonder why it came back with a `404`...
http://onemohrti.me/uses